tables/basic-table/step3.md

```diff
@@ -0,0 +1,46 @@
+Now that you have the basic table structure and imports in place, let's express column and row data as objects. This will allow us to attach and adjust data about individual rows/cells/columns later on.
+
+<strong>Note: </strong> You can mix string and object definitions for your table data.
+
+## Task
+
+1) For each column header, replace the string with an object and set the object’s title property to the string value you had previously
+
+2) For each item in your row definition, replace the string array with an array of objects, where the cell property of the object is the row array you had previously.
```

Comment:
I can figure out what you're asking me to do, by looking at the final code, but prefer you show me at each step along the way.

For example, 

1. Locate the following code 

`const columns = [...];`

2. Replace it with...

```
const columns = [
  { title: "First column" },
  { title: "Second column" },
  { title: "Third column" }
];
```